utilities/transactions/pessimistic_transaction_db.h

```diff
@@ -450,10 +442,10 @@ class WritePreparedTxnDB : public PessimisticTransactionDB {
   std::atomic<bool> delayed_prepared_empty_ = {true};
   // Update when old_commit_map_.empty() changes. Expected to be true normally.
   std::atomic<bool> old_commit_map_empty_ = {true};
-  port::RWMutex prepared_mutex_;
-  port::RWMutex old_commit_map_mutex_;
-  port::RWMutex commit_cache_mutex_;
-  port::RWMutex snapshots_mutex_;
+  mutable port::RWMutex prepared_mutex_;
```

Comment:
why mutable?

Reply:
I'm adding `const` keyword to `IsInSnapshot` call and `mutable` here means holding mutex doesn't violate `const`.